src/custom/components/Settings/SettingsMod.tsx

```diff
@@ -147,18 +147,18 @@ export default function SettingsTab({ className, SettingsButton }: SettingsTabPr
           <AutoColumn gap="lg">
             <RowBetween style={{ padding: '0 2rem' }}>
               <div />
-              <Text fontWeight={500} fontSize={20}>
+              <Text fontWeight={500} fontSize={20} color={theme.text2}>
```

Comment:
Some complexity around targeting the <Text> component specifically (to override/set the `color` CSS property). Just added the color prop here in the Mod file. Let me know if we need to change.

Reply:
its fine